sport/app/football/controllers/model.scala

```diff
@@ -37,8 +37,10 @@ class FootballPage(
    * This is a temporary solution:
    * until football feed pages make a call to capi to get branding data,
    * this has to be hardcoded here.
+   *
+   * Has to be 'def' to pick up current switch state.
```

Comment:
I'm bound to change it to a `val` the next time I look at this code, without thinking about the consequences.  The comment is to save me from myself.